### ISSUE TITLE
'end' is emitted on pause, not on completion

### ISSUE THREAD

**Issue by jmwilkinson** [isaacs/node-glob#323]
As the title says, if you listen for the end event, it will be emitted if the Glob object is paused. It will be emitted only once, on the first pause. It will not be emitted when the Glob object completes.

**Comment by bitstrider** [comment]
This seems to happen sometimes with certain configurations and directory structures.  I'm currently working on a PR with a test case for this bug, will post soon.

**Comment by bitstrider** [comment]
#328

**Comment by isaacs** [comment]
No longer relevant in v9
